Added necessary info to run the docker

petr-tik opened the pull request:
expose the default flask port to the host machine

run.py not present yet, will be run

petr-tik committed:
Added necessary info to run the docker

expose the default flask port to the host machine

run.py not present yet, will be run

ajbennieston commented on Dockerfile:
```diff
@@ -16,3 +16,6 @@ ADD https://github.com/FFIG/ffig/archive/master.zip /home/ffig/ffig
 COPY . /home/ffig/rest-api
+WORKDIR /home/ffig/flask
+EXPOSE 5000
+RUN python ./run.py
```
If it isn't present yet, let's add this line when it is present.

ajbennieston replied:
Also, is RUN the right thing? I thought RUN was to issue commands inside the image while building it; it sounds like this is the entry point when starting a container. I think there's a different command for that?

petr-tik replied:
you are right, closed it

ajbennieston replied:
Apologies for any confusion - I was happy with the `WORKDIR` and `EXPOSE` lines; it was only the `RUN` line I wanted to ask about.

ajbennieston committed:
Remove the RUN line

The WORKDIR and EXPOSE lines are fine, but RUN isn't the right command for this, and we don't have the run.py script yet anyway.

ajbennieston commented:
Good to go.

ajbennieston commented:
Looks like this repo isn't set up to `merge --squash`...
